webapp/loggedin/modules/analysis/processingChains/processingChainsView.js

```diff
@@ -11,15 +11,16 @@ import TableView from '../../../tableViews/tableView'
 
 import ProcessingChainsRow from './components/processingChainsRow'
 import ProcessingChainsRowHeader from './components/processingChainsRowHeader'
+import ProcessingChainsHeaderLeft from './components/processingChainsHeaderLeft'
 
 import * as SurveyState from '../../../../survey/surveyState'
 
 import { reloadListItems } from '../../../tableViews/actions'
-import { appModuleUri, analysisModules } from '../../../appModules'
+import { createProcessingChain, navigateToProcessingChainView } from './actions'
 
 const processingChainsModule = 'processing-chains'
 
-const ProcessingChainsView = ({ surveyCycleKey, reloadListItems, history }) => {
+const ProcessingChainsView = ({ surveyCycleKey, reloadListItems, history, createProcessingChain, navigateToProcessingChainView }) => {
 
```

Comment:
it would be more readable if you use props:
```
const ProcessingChainsView = props => {
    const { 
        surveyCycleKey, history, 
        reloadListItems,  createProcessingChain, navigateToProcessingChainView
     } = props
....
```

Reply:
ok